src/AWSS3.jl

```diff
@@ -904,7 +925,9 @@ function s3_purge_versions(
     end
 end
 
-s3_purge_versions(a...; b...) = s3_purge_versions(global_aws_config(), a...; b...)
+function s3_purge_versions(bucket, path_prefix="", pattern=""; kwargs...)
+    return s3_purge_versions(global_aws_config(), bucket, path_prefix, pattern; kwargs...)
+end
```

Comment:
Avoid redefining default values:
```suggestion
function s3_purge_versions(args...; kwargs...)
    return s3_purge_versions(global_aws_config(), args...; kwargs...)
end
```